### ISSUE TITLE
find-and-replace width does not update when window size is decreased

### ISSUE THREAD

**Issue by Yajo** [atom/find-and-replace#732]
![captura de pantalla de 2016-05-25 13-19-09](https://cloud.githubusercontent.com/assets/973709/15538306/a27b1740-227b-11e6-912c-67b69cd9911d.png)

## Repro Steps

1. Right-click on any folder in the tree view.
2. Press *Search in Directory*.
3. Increase the text editor panel width.
4. Decrease it.

**Expected:** Find & replace panel matches the text editor panel's width.
**Actual:** Find & replace panel overflows itself and the above panel.

It happens with `atom --safe` too.

## Versions

* **Atom:**       1.7.3
* **Atom-Shell:** 
* **OS:**         linux 4.4.9-300.fc23.x86_64
* **Misc**
    * apm  1.10.0
    * npm  1.3.6
    * node 0.10.42
    * python 2.7.11
    * git 2.5.5

---

<small>This report was created in and posted from the Atom editor using the package `bug-report` v0.7.1.</small>

**Comment by adambuczynski** [comment]
This also affects non-folder searches. See https://github.com/atom/find-and-replace/issues/734

**Comment by simurai** [comment]
I can reproduce this in Atom `1.9.0-dev-5eb0a03`. Same version as in #734.

It looks like all `atom-text-editor.mini` don't resize anymore. You can also test it when resizing the settings. Growing works, but shrinking not.

When digging a bit deeper, it looks like the inlined `width` of the `.lines` element doesn't get updated when reducing the window width. It stays at the same value.

![screen shot 2016-05-28 at 3 46 04 pm](https://cloud.githubusercontent.com/assets/378023/15625725/391f9d60-24ed-11e6-94d7-7d66fad0417e.png)

A temporary hack is to just override the `width` with `!important` like:

```css
atom-text-editor.mini::shadow .lines {
  width: auto !important;
}
```

But that probably shouldn't be the final fix.

@as-cii do you know where the `width` gets updated and if something related changed recently?

**Comment by Yajo** [comment]
In which version is this fix? I'm starting to pull out my hair just because of this bug...

**Comment by 50Wliu** [comment]
This fix is not yet in any released version.

**Comment by adambuczynski** [comment]
@Yajo you can enjoy the fix by building the latest version of Atom yourself from the master branch. 
See https://github.com/atom/atom/tree/master/docs/build-instructions

**Comment by simurai** [comment]
@Yajo It's fixed in [1.9](https://github.com/atom/atom/releases/tag/v1.9.0-beta0). So it should already be in [Atom Beta](https://atom.io/beta).
